pkg/solana/config/toml.go

```diff
@@ -285,7 +286,7 @@ func (c *TOMLConfig) ListNodes() Nodes {
 	return c.Nodes
 }
 
-func (c *TOMLConfig) MultiNodeConfig() *MultiNode {
+func (c *TOMLConfig) MultiNodeConfig() *MultiNodeConfig {
 	return &c.MultiNode
 }
```

Comment:
is this still needed if we just use `c.MultiNode`?

Reply:
This wrapper is needed in order to allow the config names to be public, while also having the required functions names which are the same.